skimage/measure/_regionprops_utils.py

```diff
@@ -326,3 +327,38 @@ def perimeter_crofton(image, directions=4):
 
     total_perimeter = coefs @ h
     return total_perimeter
+
+
+def _normalize_spacing(spacing, ndims):
+    """Normalizes the spacing parameter
+
+    The `spacing` parameter should be a tuple of numbers
+    that is the same length as the number of dimensions
+    in the image.
+    If the spacing is a single number (e.g. float)
+    then assume spacing is same in all dimensions.
+
+    Parameters
+    ---------
+    spacing : Any
+        User-provided `spacing` keyword.
+    ndims: int
+        The number of dimensions in the image
+
+    Returns
+    -------
+    spacing : tuple of floats
```

Comment:
```suggestion
    spacing : array
```
If `spacing` is an integer, it doesn't get converted into a float, does it?